lib/utils.ts

```
@@ -1,45 +1,237 @@
+  let normalizedType = errorType;
+  let normalizedMessage = errorMessage;
+  let normalizedCategory = category || 'unknown_error';
+  let groupKey = '';
+
+  // Only categorize webhook errors and stalled errors, keep everything else as-is
+  
+  // Webhook error normalization - simply group by status code
+  if (category === 'webhook_error' && errorMessage.includes('Status:')) {
+    const statusMatch = errorMessage.match(/Status: (\d+)/);
+    if (statusMatch && statusMatch[1]) {
+      const statusCode = parseInt(statusMatch[1], 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)